src/Core/src/PublicAPI/net-ios/PublicAPI.Shipped.txt

```diff
@@ -1112,7 +1112,6 @@ Microsoft.Maui.IStackNavigation.NavigationFinished(System.Collections.Generic.IR
 Microsoft.Maui.IStackNavigation.RequestNavigation(Microsoft.Maui.NavigationRequest! eventArgs) -> void
 Microsoft.Maui.IStackNavigationView
 Microsoft.Maui.IStepper
-Microsoft.Maui.IStepper.Interval.get -> double
```

Comment:
To fix these errors you have to add a *Removed* line into the UNShipped.txt file

for example, 
https://github.com/dotnet/maui/blob/934e1879b12c7b2dc248a41f87cd7734382d88c2/src/Core/src/PublicAPI/net-ios/PublicAPI.Unshipped.txt#L212-L218

Reply:
I have added the required 'Removed' line to the UnShipped.txt file and pushed the changes. Please review the changes and let me know if there are any additional adjustments needed.